tests/integrations/resource_metering/test_dynamic_config.rs

```diff
@@ -76,15 +76,20 @@ pub fn test_report_interval() {
     // | Report Interval |
     // |       1s        |
     test_suite.cfg_report_receiver_interval("1s");
-    let res = test_suite.block_receive_one();
-    let begin = Instant::now();
-    assert!(res.contains_key("req-1"));
```

Comment:
Why were these two assertions removed?

Reply:
This test doesn't care about this. It was just stricter before.